.eslintrc

```diff
@@ -688,4 +688,4 @@ rules:
   no-bitwise: 0
 
   # Disallows use of unary operators, ++ and --.
-  no-plusplus: 0
+  no-plusplus: 0
```

Comment:
@imuchnik, I think it would be valuable to test that no listeners are registered after an element has been destroyed/removed.